### PR TITLE
ID Access Fixes

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [New] There's a low chance that you can assume the identity of someone by holding up their ID card in your active hand while naked.
CL: [New] Items with `IClearanceSource` inside storage containers can now be checked for clearance.
CL: [Fix] Fixed an issue with the `ClearanceRestricted` API blocking access checks for cyborgifed players who don't have access on their new bodyParts.
CL: [Fix] `ClearanceRestricted` will always check clothing first now before anything else.
CL: [Fix] Belts and suit storages were not checked at all before for clearance.
CL: [Fix] Fixed an issue with player names reporting "unknown" despite the existence of an ID somewhere on the player.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/Clearance/ClearanceRestricted.cs [hunk 1]
```diff
@@ -139,5 +103,55 @@ public void SetCheckType(CheckType newType)
...
+		{
+			if (entity == null)
+			{
+				return null;
+			}
+			var playerStorage = entity.GetComponent<DynamicItemStorage>();
+			if (playerStorage != null)
+			{
+				List<ItemSlot> slotsToSearch = new List<ItemSlot>();
+				slotsToSearch.Add(playerStorage.GetActiveHandSlot());
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Systems/Clearance/ClearanceRestricted.cs]: surely best just to grab all slots?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/Clearance/ClearanceRestricted.cs]: Updated it to grab all hand slots instead.

**Comment by Bod9001** [comment]
I mean all slots in your inventory?

**Comment by MaxIsJoe** [comment]
> I mean all slots in your inventory?

That isn't the intended design, and doesn't make sense a lot.
Your phone is not visible inside your pockets, but if you holster it on your belt or breast pocket; it is very much visible.

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**